resources/views/checkout/partials/sections/address.blade.php

```diff
@@ -1,9 +1,25 @@
-<section>
-    <form id="credentials" v-on:submit.prevent="save(['credentials'], 2)">
+<checkout-address v-slot="{ useCards, editing, popup, toggleEdit, togglePopup, hideBilling, isType, select }">
+    <section v-if="useCards && !editing">
+        @include('rapidez-ct::checkout.partials.address-cards')
+        <div class="mt-5">
+            <x-rapidez-ct::button.accent v-on:click.prevent="toggleEdit">
+                @lang('Use a new address')
+            </x-rapidez-ct::button.accent>
+            <x-rapidez-ct::button.outline v-on:click.prevent="togglePopup">
+                @lang('My addresses')
+            </x-rapidez-ct::button.outline>
```

Comment:
should we be consistent with toggling elements or with vue or with plain css by peer-checking input checkboxes?
this is now being done with JS, while the slide-over component made by Jimmy is made with css